tests/c_languages/CPPCleanBearTest.py

```diff
@@ -1,5 +1,6 @@
 from bears.c_languages.CPPCleanBear import CPPCleanBear
 from coalib.testing.LocalBearTestHelper import verify_local_bear
+import os
```

Comment:
please put the import at top of this file, although we can't enforce it yet (with coala), our yet inofficial rule is:
```
system-imports

own imports

other package imports
```